src/Components/Bookings/BookingsModal/BookingsModal.css

```diff
@@ -0,0 +1,121 @@
+.booking-modal {
+	position: fixed;
+	background: white;
+	box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.26);
+	top: 30vh;
+	width: 20rem;
+	left: 2rem;
+	word-spacing: 2rem;
+}
+.booking-modal span label {
```

Comment:
I see this convention a lot in this PR and I'm not sure what the reasoning is. It has **parent class and then HTML elements**.
- It makes the classes be bloated and perhaps super hard to manage.
- It's generally a bad practice to style HTML element directly. What if you add another span or label tomorrow? You'd need to revisit this again and hack your way around it.

My recommendation to clean up your CSS a little more
```css
.booking-modal {}

.booking-modal__button { }

.booking-actions { }

.booking-actions__button { }

```

This uses [BEM](http://getbem.com/introduction/) to flatten the styles and with SCSS or POSTCSS it even gets more interested because you can do things like

```css
.booking-modal {
   &__button { }
   
    &__text { }
}

BTW, this can be left as-is but it's important we highlight the CSS needs some structuring

```